requirements.txt

```diff
@@ -47,4 +47,5 @@ tabulate
 torchvision==0.5.0
 tqdm
 h5py
-pretrainedmodels==0.7.4
\ No newline at end of file
+pretrainedmodels==0.7.4
+wandb
```

Comment:
Could you add the version you tested with?

Reply:
You got it. Added.